odata-csdl/14 Vocabulary and Annotation.md

```diff
@@ -1757,13 +1755,11 @@ types](#BuiltInTypesfordefiningVocabularyTerms)
 `Edm.PropertyPath`, `Edm.AnyPropertyPath`, or `Edm.ModelElementPath`. Its
 argument is a [model path](#PathExpressions) with the following
 restriction:
-- A non-null path MUST resolve to a model
-element whose type is a primitive or complex type, an enumeration type,
+- A non-null path MUST end with a [structural property](#StructuralProperty)
+or a term cast to a term whose type is a primitive or complex type, an enumeration type,
 a type definition, or a collection of one of these types.
 
-The value of the property path expression is the path itself, not the
-value of the structural property or the value of the term cast
-identified by the path.
+The value of the property path expression is the structural property or term cast, not its instances.
```

Comment:
```suggestion
The value of the property path expression is the structural property (not its instances) or the term (not the annotation with that term).
```

Reply:
What I want to express: given a type
~~~xml
<EntityType Name="Foo">
  <Annotation Term="Common.SortOrder">
    <Collection>
      <Record>
        <PropertyValue Property="Property" PropertyPath="Amount/@Measures.ISOCurrency"/>
      </Record>
      <Record>
        <PropertyValue Property="Property" PropertyPath="Amount"/>
      </Record>
    </Collection>
  </Annotation>
  <!-- ... -->
</EntityType>
~~~

Then the value of the `Common.SortOrder` annotation is
~~~json
[
  {"Property":"Amount/@Measures.ISOCurrency"},
  {"Property":"Amount"}
]
~~~
and the values of the two `PropertyPath` expressions are `Amount/@Measures.ISOCurrency` and `Amount`.

This is what I meant with 
> The value of the property path expression is the structural property or term cast

The client would use the property and term cast _names_ to construct 
~~~
$orderby=Amount/@Measures.ISOCurrency,Amount
~~~
and would not be interested in the actual instance-specific values.

Replacing "term cast" with "annotation" would confuse at least me, and I am open for a better phrase.

Reply:
I am confused by "term cast", because a model element path must resolve to a model element. To be discussed in TC.

Reply:
Properties and annotations can be used interchangeably, that is (instance) annotations are a comparable to dynamic properties.

How about
```suggestion
The value of the property path expression is the path to the structural property or annotation, not its instance value.
```

Reply:
I like that, let's discuss in TC.

Reply:
Aligned with V4.01 text and replaced "term cast" with "annotation".